zuul.d/project-templates.yaml

```diff
@@ -116,6 +118,8 @@
               - name: github.com/ansible-collections/ansible.utils
               - name: github.com/ansible-collections/community.general
         - ansible-test-cloud-integration-aws-py36
+        - ansible-galaxy-importer:
+            voting: false
```

Comment:
lets remove non-voting from gate, since results are not used for merging

Reply:
I'm not sure I understand. `ansible-galaxy-importer` job also runs during the gating.

Reply:
non-voting jobs in gate, are a waste of resources. So, rather making a job non-voting, removing it is better. And nice on nodepool.

Reply:
The job is pulled by `publish-to-galaxy` and I don't want to change project-config just for this temporary fix. I don't see any easy way to disable a job?